python.d/httpcheck.chart.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+# Description: http check netdata python.d module
+# Original Author: ccremer (github.com/ccremer)
+
+import urllib3
+import time
+import re
+
+from bases.FrameworkServices.UrlService import UrlService
+
+try:
+    urllib3.disable_warnings()
+except AttributeError:
+    pass
+
+# default module values (can be overridden per job in `config`)
+# update_every = 2
```

Comment:
Default `update_every` is 1. Do we need 1 seconds `update_every` as default value for this plugin?

Reply:
Personally, I would set it to 3 seconds in the config (it's what I have now in my environment). But since netdata's default is 1 secs, I defaulted to this default ;) Would you like to change it?

Reply:
> netdata's default is 1 secs

Depends on module. 
I am not sure someone need to "monitor" http status every second. The other problem is delays (network delay 2-way, request process delay, response process delay).
5 seconds looks ok for me.

Lets summon @ktsaou and @Ferroin

Reply:
I agree that 1 sec is too much, but increasing the interval results in lower resolution for averaging response times and for alarms. The current alarm templates fire the alarm after 30 checks of subsequent failures with 1sec resolution. With 5 sec we would currently fire the alarm after 6 checks. I don't know if that would increase false positives.